operations/repo.go

```diff
@@ -46,7 +84,55 @@ func Repo() cli.Command {
 	}
 }
 
-func repoFlags() []cli.Flag {
+func repoSubmit() cli.Command {
+	return cli.Command{
+		Name:  "submit",
+		Usage: "submit a repobuilder job to a remote service",
+		Flags: repoFlags(
+			cli.StringSliceFlag{
+				Name:  "packages",
+				Usage: "path to packages, searches for valid packages recursively",
+			},
+			cli.StringFlag{
+				Name:  "service",
+				Usage: "specify the path to a repobuilder service",
+				Value: "https://barque.mongodb.comm",
```

Comment:
"comm", also should we really hard code something like this?

Reply:
it's just a default, eh?